docs/access_and_permissions.md

```diff
@@ -302,13 +412,15 @@ flowchart TD
 
 ### Controllers
 
-| Class                                                                                  | Responsibility                                                                     |
-|----------------------------------------------------------------------------------------|------------------------------------------------------------------------------------|
-| [Account::ApplicationsController](/app/controllers/account/applications_controller.rb) | Managing own access to apps and accessing own permissions routes                   |
-| [Account::PermissionsController](/app/controllers/account/permissions_controller.rb)   | Managing own permissions                                                           |
-| [InvitationsController](/app/controllers/invitations_controller.rb)                    | Giving users access to apps and permissions while creating their account           |
-| [Users::ApplicationsController](/app/controllers/users/applications_controller.rb)     | Managing other users' access to apps and accessing other users' permissions routes |
-| [Users::PermissionsController](/app/controllers/users/permissions_controller.rb)       | Managing other users' permissions                                                  |
+| Class                                                                                             | Responsibility                                                                     |
+| ------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------- |
+| [Account::ApplicationsController](/app/controllers/account/applications_controller.rb)            | Managing own access to apps and accessing own permissions routes                   |
+| [Account::PermissionsController](/app/controllers/account/permissions_controller.rb)              | Managing own permissions                                                           |
+| [Account::SigninPermissionsController](/app/controllers/account/signin_permissions_controller.rb) | Handles updating of own permissions when granting or removing access               |
```

Comment:
This could just be `Managing own access to apps` like the applications controller above, but I'm not fussed either way. (And `Managing other users' access to apps` for the users equivalent)